cmd/admin-revoker/main.go

```diff
@@ -130,6 +130,21 @@ func revokeByReg(regID int64, reasonCode core.RevocationCode, deny bool, rac rpc
 	return
 }
 
+// This abstraction is needed so that we can use sort.Sort below
```

Comment:
I don't see any call to sort.Sort in this file. And if there were, it would be a functionality change, not a lint cleanup, so would belong in a different PR.